.nvmrc

```diff
@@ -0,0 +1 @@
+v21.7.3
```

Comment:
is this your own nvmrc? ill have to check with infra if theyre ok with adding this configuration.

do you think this is a necessary change?

Reply:
yeah, that is my `.nvmrc` file
With that file you do not need to think which nodejs version should be used for that project  - you just run `nvm use` - and you have the right version... with nvm usage, life become easier

Reply:
heya @mihailShumilov sorry for the delay. Just got one request. Could you remove the nvmrc? 
thank you.